command/sub8_alarm/nodes/alarm_handler.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+import rospy
+import json
+from sub8_msgs.msg import Alarm
+from std_msgs.msg import Header
+from sub8_alarm import alarms
+import string
+
+
+class AlarmHandler(object):
+    def __init__(self):
+        '''Alarm Handler
+            Listen for alarms, call scenarios
+        TODO:
+            - Add alarm queue
+            - Handle alarms in sequence (Don't get stuck in the callback)
+            - bag (if set) EVERY single alarm received
+        '''
+        rospy.init_node('alarm_handler')
+        # Queue size is large because you bet your ass we are addressing every alarm
+        self.alarm_sub = rospy.Subscriber('/alarm', Alarm, self.alarm_callback, queue_size=100)
```

Comment:
We should monitor this queue size value as the system progresses- it may prove to be too small if multiple things fail at once, and lost alarms would be sad